concordium-node/src/consensus_ffi/ffi.rs

```diff
@@ -241,7 +246,7 @@ type LogCallback = extern "C" fn(c_char, c_char, *const u8);
 type BroadcastCallback = extern "C" fn(i64, *const u8, i64);
 type CatchUpStatusCallback = extern "C" fn(*const u8, i64);
 type DirectMessageCallback =
-    extern "C" fn(peer_id: PeerId, message_type: i64, msg: *const c_char, msg_len: i64);
+    extern "C" fn(peer_id: PeerIdFFI, message_type: i64, msg: *const c_char, msg_len: i64);
```

Comment:
Unrelated to changes: Why is msg_len not u64?

Reply:
No really interesting reason. In Haskell lengths/sizes are (for historical reasons) always Int. I imagine that is the source.